examples/complete/variables.tf

```diff
@@ -0,0 +1,174 @@
+##############################################################################
```

Comment:
The example variables should only contain the minimum. The default values in the variable should be put inline in the main.tf of the example with some explanations.

Reply:
Done.